README.md

```diff
@@ -46,7 +46,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath("org.springframework.boot:spring-boot-gradle-plugin:1.5.10.RELEASE")
+        classpath("org.springframework.boot:spring-boot-gradle-plugin:2.1.6.RELEASE")
```

Comment:
I think this should be `implementation` not `classpath`.. According to the official [Dependency Management for Java Projects](https://docs.gradle.org/current/userguide/dependency_management_for_java_projects.html).

Reply:
Oh, my bad.. it is actually a build script dependency.. not a java dependency. All good 👍